internal/constants/constants.go

```diff
@@ -137,6 +137,8 @@ const (
 	EventReasonGetObject            = "GetObject"
 	EventReasonRemoveObject         = "RemoveObject"
 	EventReasonUpdateObject         = "UpdateObject"
+
+	MigrateDefaultOutput = "default-replaced"
```

Comment:
Looks like we maybe will have to rebase on @cahartma PR?

Reply:
Yes will need to rebase